### PR TITLE
[readme] refactor readme file

### PR DESCRIPTION
**Pull request by delhomer**
Highlight most important informations like project context and
description, how to install it, and design a clearer example. Remove
the motivation part with the urbansprawl figures (we can expect that
interested readers will follow the indicated references for detailed
data about this phenomenon).

:warning: Add upstream project information -> from now, we consider Oslandia/urbansprawl as the reference project where to push PRs (replace delhomer/urbansprawl)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,31 @@
-# Urbansprawl
+# About Urbansprawl
 
-The urbansprawl project provides an open source framework for assessing urban sprawl using open data. It uses OpenStreetMap (OSM) data to calculate its sprawling indices, divided in Accessibility, Land use mix, and Dispersion.
+The **urbansprawl** project takes its origin from a PhD thesis that was
+defended in november 2018 by Luciano Gervasoni at Inria. Oslandia takes over
+from Inria fellows for industrial developments, and now hosts the upstream
```
Comment (garaud) [inline README.md]: can you write "now hosts the upstream version" in bold?

**Review by garaud — CHANGES_REQUESTED** [review]
just a tiny comment

after that, good for me

**Comment by garaud** [comment]
and maybe add the link to the previous upstream github project

File: README.md (hunk 1 above)
Comment (delhomer) [reply README.md]: Done.

**Review by garaud — APPROVED** [review]
:+1:

**Pull request merged**